src/server/rooms/NetsBloxSocket.js

```diff
@@ -373,6 +373,12 @@ NetsBloxSocket.MessageHandlers = {
                 });
             });
         }
+    },
+
+    'request-new-name': function(msg) {
```

Comment:
probably should remove `msg` from the list of arguments. This should throw an "unused-variable-warning" in jshint/eslint